.env.example

```diff
@@ -101,4 +105,10 @@ SYNCHRONIZER_DUMP_SYNC_THRESHOLD=50000000
 SYNCHRONIZER_PARALLEL_TASKS=30
 
 # Profiling config
-MALLOC_CONF="prof:true,prof_leak:true,prof_final:true,prof_active:true,prof_prefix:/usr/src/app/heaps/,lg_prof_interval:32,lg_prof_sample:19"
\ No newline at end of file
+MALLOC_CONF="prof:true,prof_leak:true,prof_final:true,prof_active:true,prof_prefix:/usr/src/app/heaps/,lg_prof_interval:32,lg_prof_sample:19"
```

Comment:
this env is optional
you need to indicate it only when we want to run memory profiling